lib/libsomeip-c/example/test_service.cc

```diff
@@ -0,0 +1,138 @@
+/*  Fluent Bit
```

Comment:
This seems to be a C++ file, is that just a standard example and/or can it be C only? It seems to require a C++ compiler by looking at the example cmake linkage.